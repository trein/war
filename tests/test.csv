Ad ID,Impression,Click,Conv,Bw Impression,Bw Click,Bw Conv
8073,8299.0,144.0,2.0,4.0,0.0,0.0
3605,861.0,35.0,21.0,861.0,35.0,21.0
3603,1045.0,19.0,29.0,1045.0,19.0,29.0
8105,4949.0,87.0,5.0,113.0,2.0,0.0
8216,3940.0,59.0,11.0,65.0,2.0,0.0
3592,807.0,16.0,11.0,807.0,16.0,11.0
8317,16644.0,269.0,10.0,10447.0,166.0,7.0
3608,992.0,19.0,24.0,992.0,19.0,24.0
3604,806.0,20.0,15.0,806.0,20.0,15.0
8069,6744.0,88.0,1.0,4.0,0.0,0.0
8102,17891.0,360.0,42.0,17093.0,346.0,42.0
3325,7606.0,66.0,0.0,589.0,4.0,0.0
8294,8699.0,34.0,85.0,4656.0,21.0,46.0
3607,810.0,16.0,10.0,810.0,16.0,10.0
8101,22619.0,466.0,69.0,19051.0,371.0,63.0
8071,6644.0,91.0,13.0,4174.0,62.0,9.0
3328,1898.0,13.0,0.0,27.0,3.0,0.0
8021,5277.0,389.0,9.0,2331.0,147.0,6.0
8390,271.0,3.0,8.0,0.0,0.0,0.0
8212,1775.0,28.0,5.0,374.0,9.0,4.0
8384,0.0,0.0,2.0,0.0,0.0,0.0
8020,2017.0,17.0,0.0,2.0,0.0,0.0
3332,648.0,3.0,0.0,93.0,0.0,0.0
8298,22457.0,288.0,45.0,866.0,10.0,1.0
3594,1017.0,42.0,20.0,1017.0,42.0,20.0
8388,6744.0,114.0,9.0,177.0,1.0,0.0
3517,16718.0,154.0,183.0,16718.0,154.0,183.0
8362,1335.0,8.0,0.0,3.0,0.0,0.0
3595,569.0,20.0,11.0,569.0,20.0,11.0
8389,4181.0,81.0,7.0,5.0,0.0,0.0
8296,2658.0,6.0,9.0,315.0,0.0,0.0
8300,6962.0,106.0,17.0,5.0,0.0,0.0
8412,13857.0,149.0,12.0,13773.0,146.0,11.0
3597,615.0,8.0,6.0,615.0,8.0,6.0
3323,30731.0,310.0,8.0,2737.0,23.0,6.0
8304,45882.0,658.0,100.0,1320.0,27.0,19.0
8336,4096.0,46.0,60.0,4096.0,46.0,60.0
8323,1802.0,22.0,26.0,1802.0,22.0,26.0
8406,2500.0,59.0,3609.0,2500.0,59.0,3609.0
8405,2494.0,56.0,3842.0,2494.0,56.0,3842.0
8387,21964.0,285.0,35.0,172.0,3.0,0.0
3596,729.0,10.0,2.0,729.0,10.0,2.0
3606,1312.0,45.0,52.0,1312.0,45.0,52.0
8324,1915.0,17.0,21.0,1915.0,17.0,21.0
8401,23798.0,270.0,11.0,6326.0,98.0,9.0
3593,699.0,18.0,16.0,699.0,18.0,16.0
8106,6369.0,102.0,9.0,5577.0,86.0,9.0
8322,1894.0,22.0,31.0,1894.0,22.0,31.0
3389,2604.0,45.0,46.0,2604.0,45.0,46.0
