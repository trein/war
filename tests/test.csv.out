cu8105,4949.0,87.0,5.0,113.0,2.0,0.0cu
cu8317,16644.0,269.0,10.0,10447.0,166.0,7.0cu
cu3608,992.0,19.0,24.0,992.0,19.0,24.0cu
cu3604,806.0,20.0,15.0,806.0,20.0,15.0cu
cu8101,22619.0,466.0,69.0,19051.0,371.0,63.0cu
cu3328,1898.0,13.0,0.0,27.0,3.0,0.0cu
cu8390,271.0,3.0,8.0,0.0,0.0,0.0cu
cu3332,648.0,3.0,0.0,93.0,0.0,0.0cu
cu3594,1017.0,42.0,20.0,1017.0,42.0,20.0cu
cu3323,30731.0,310.0,8.0,2737.0,23.0,6.0cu
cu8387,21964.0,285.0,35.0,172.0,3.0,0.0cu
cu8401,23798.0,270.0,11.0,6326.0,98.0,9.0cu
cu8106,6369.0,102.0,9.0,5577.0,86.0,9.0cu
cu3389,2604.0,45.0,46.0,2604.0,45.0,46.0cu
cu3605,861.0,35.0,21.0,861.0,35.0,21.0cu
cu8071,6644.0,91.0,13.0,4174.0,62.0,9.0cu
cu8020,2017.0,17.0,0.0,2.0,0.0,0.0cu
cu8388,6744.0,114.0,9.0,177.0,1.0,0.0cu
cu3595,569.0,20.0,11.0,569.0,20.0,11.0cu
cu8323,1802.0,22.0,26.0,1802.0,22.0,26.0cu
cu8406,2500.0,59.0,3609.0,2500.0,59.0,3609.0cu
cuAd ID,Impression,Click,Conv,Bw Impression,Bw Click,Bw Convcu
cu3592,807.0,16.0,11.0,807.0,16.0,11.0cu
cu8069,6744.0,88.0,1.0,4.0,0.0,0.0cu
cu8102,17891.0,360.0,42.0,17093.0,346.0,42.0cu
cu8294,8699.0,34.0,85.0,4656.0,21.0,46.0cu
cu8212,1775.0,28.0,5.0,374.0,9.0,4.0cu
cu8384,0.0,0.0,2.0,0.0,0.0,0.0cu
cu8362,1335.0,8.0,0.0,3.0,0.0,0.0cu
cu8389,4181.0,81.0,7.0,5.0,0.0,0.0cu
cu8296,2658.0,6.0,9.0,315.0,0.0,0.0cu
cu8405,2494.0,56.0,3842.0,2494.0,56.0,3842.0cu
cu3596,729.0,10.0,2.0,729.0,10.0,2.0cu
cu3606,1312.0,45.0,52.0,1312.0,45.0,52.0cu
cu8324,1915.0,17.0,21.0,1915.0,17.0,21.0cu
cu8322,1894.0,22.0,31.0,1894.0,22.0,31.0cu
cu8073,8299.0,144.0,2.0,4.0,0.0,0.0cu
cu3603,1045.0,19.0,29.0,1045.0,19.0,29.0cu
cu8216,3940.0,59.0,11.0,65.0,2.0,0.0cu
cu3325,7606.0,66.0,0.0,589.0,4.0,0.0cu
cu3607,810.0,16.0,10.0,810.0,16.0,10.0cu
cu8021,5277.0,389.0,9.0,2331.0,147.0,6.0cu
cu8298,22457.0,288.0,45.0,866.0,10.0,1.0cu
cu3517,16718.0,154.0,183.0,16718.0,154.0,183.0cu
cu8300,6962.0,106.0,17.0,5.0,0.0,0.0cu
cu8412,13857.0,149.0,12.0,13773.0,146.0,11.0cu
cu3597,615.0,8.0,6.0,615.0,8.0,6.0cu
cu8304,45882.0,658.0,100.0,1320.0,27.0,19.0cu
cu8336,4096.0,46.0,60.0,4096.0,46.0,60.0cu
cu3593,699.0,18.0,16.0,699.0,18.0,16.0cu
